packages/gestalt/src/ButtonGroup.tsx

```diff
@@ -1,5 +1,7 @@
 import { Children, ReactNode } from 'react';
+import { Flex } from '.';
```

Comment:
Can we do this change in a separate PR? Just ButtonGroup?

Reply:
https://github.com/pinterest/gestalt/pull/3840